<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphiteOptionsWidgetUI</class>
 <widget class="QWidget" name="GraphiteOptionsWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Area Fill Mode:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mAreaModeCombo">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>First</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stacked</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Time Range:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="mTimeRangeCombo">
     <item>
      <property name="text">
       <string>Last Day</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last Week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last Month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last Year</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Specify Range...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="mTimeRangeGroup">
     <property name="title">
      <string>Specify Time Range</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="mStartDateTimeEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>End:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="mEndDateTimeEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Value Range:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="mValueRangeCombo">
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Percentage(0-100)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Specify Range...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="mValueRangeGroup">
     <property name="title">
      <string>Specify Value Range</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="mMinValueSpin">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="mMaxValueSpin">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
    <property name="text">
      <string>Extra Options:</string>
    </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
       <widget class="QListWidget" name="mExtraList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
